frame/staking/src/pallet/mod.rs

```diff
@@ -124,6 +125,25 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxNominations: Get<u32>;
 
+		/// Number of eras to keep in history.
+		///
+		/// Information is kept for eras in `[current_era - history_depth;
```

Comment:
what is "information"?

answer: `Ledger.claimed_rewards`, and a number of era information such as `ErasStakers`, as we discussed.

Reply:
https://github.com/paritytech/substrate/pull/12230/commits/121557df70409447c70c548256e15fe4626a3f6e